.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
OK, shall we add this to the checklist, then? 
 - [ ] Branch rebased on top of master
 - [ ] Changes squashed to a single commit


... and I'll add this to the contributor guide:

"Please rebase your pull request on top of the current master (`git pull --rebase origin master`) and squash your changes to a single commit as described [here](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)"